libs/ngx-charts-on-fhir/src/lib/fhir-mappers/medication-request/duration-medication-mapper.service.ts

```diff
@@ -207,12 +207,15 @@ export class DurationMedicationMapper implements Mapper<DurationMedication> {
     const layer = this.baseMapper.map(resource) as DataLayer<'bar', MedicationDataPoint[]>;
     layer.datasets = layer.datasets.map((dataset) => ({
       ...dataset,
-      label: dataset.label + ' *',
+      label: dataset.label,
       type: 'bar',
       borderWidth: 1,
       borderSkipped: false,
       barPercentage: 1,
       grouped: false,
+      chartsOnFhir: {
+        group: dataset.label,
+      },
```

Comment:
I think this is unnecessary because the base (simple) mapper is already setting the group.